### DISCUSSION TITLE
where can i write alt for photo

### DISCUSSION THREAD

**Discussion by bluesky0341** [e107inc/e107#5190] [General]
where can i write alt for photo

**Comment by bluesky0341** [comment]
<img src="https://activatedalumina.com.cn/001/activated-alumina.jpg" width="577" height="382" title="Alumina" alt="Alumina" class="img-rounded rounded bbcode bbcode-img bbcode-img-news">

**Reply by bluesky0341** [reply]
<img src="  " width="577" height="382" title="Alumina" alt="Alumina" class="img-rounded rounded bbcode bbcode-img bbcode-img-news">

**Comment by bluesky0341** [comment]
width="577" height="382" title="Alumina" alt="Alumina" class="img-rounded rounded bbcode bbcode-img bbcode-img-news"

**Reply by Jimmi08** [reply]
Try media manager for this image

**Comment by bluesky0341** [comment]
thanks 
i write activated alumina
but alt chang into alumina

**Reply by Jimmi08** [reply]
It can be set with your theme, template, used shortcode... anything. 

For `{NEWS_IMAGE} ` it looks like it is not filled...  No idea why.

```
switch($parms[1])
		{
			case 'src':
				return $src;
			break;

			case 'tag':
				return "<img class='news_image ".$class."' src='".$src."' alt='' style='".$style."' {$dimensions} {$srcset} />";
			break;

			case 'img':
				return "<a href='".$_src."' rel='external image'><img class='news_image ".$class."' src='".$src."' alt='' style='".$style."' {$dimensions} {$srcset} /></a>";
			break;

			default:
				return "<a href='".e107::getUrl()->create('news/view/item', $this->news_item)."'><img class='news_image img-responsive img-fluid img-rounded rounded ".$class."' src='".$src."' alt='' style='".$style."' {$dimensions} {$srcset} /></a>";
			break;
		}
```

**Reply by Jimmi08** [reply]
According answer to this issue, there is solution

https://github.com/e107inc/e107/issues/3927

`{NEWS_IMAGE: alt=description}`

But how to set it f.e. as news title  - I really don't know.

**Comment by bluesky0341** [comment]
thanks for reply
but this is very important function
when you upload photo,you can write alt
